src/main/java/com/cloudbees/jenkins/support/SupportPlugin.java

```diff
@@ -287,13 +287,19 @@ public static void writeBundle(OutputStream outputStream, final List<Component>
         try {
             try (BulkChange change = new BulkChange(ContentMappings.get());
                  ZipArchiveOutputStream binaryOut = new ZipArchiveOutputStream(new BufferedOutputStream(outputStream, 16384))) {
+                // Get the filter to be used
                 Optional<ContentFilter> maybeFilter = getContentFilter();
 
+                // Recalculate the mappings and stop words and save it to disk
+                if(maybeFilter.isPresent()) {
```

Comment:
Why is this necessary now?

Reply:
Before we were reloading and saving if the filter was enabled, now it's the same as the filter is not empty.